ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/PoWHasher.java

```diff
@@ -45,4 +51,20 @@ public void hash(
       System.arraycopy(hash, 0, buffer, 0, hash.length);
     }
   }
+
+  /** Implementation of an inoperative hasher. */
+  final class NoOp implements PoWHasher {
```

Comment:
This isn't a no-op, this is something else.  InvalidHasher?  None?

Reply:
I renamed to "unsupported".